apps/admin_app/assets/js/views/product/edit.js

```diff
@@ -31,6 +32,25 @@ export function imageOnEnter(){
   });
 }
 
+export function defaultImage(){
+  $(document).delegate('input[type="checkbox"]', 'click', function(event) {
+    $(this).parent().siblings().find(':checkbox').prop('checked', false);
+    var default_image = $(this).val();
+    var product_id = $("#product-id").val();
+    var data = { product_id: product_id,  default_image: default_image};
+    var CSRF_TOKEN = $("meta[name='csrf-token']").attr("content");
```

Comment:
@jyotigautam Can we please leave an empty line in between the batch of instructions such as the above is initialisation part where we are settings variable values and below this line we are sending an ajax request? What do you think?